src/tutorial/runtime/reinstalling.rst

```diff
@@ -0,0 +1,154 @@
+.. _tutorial.Reinstalling-workflows:
+
+Reinstalling Workflows
+======================
+
+.. admonition:: Aim
+
+   Use ``cylc reinstall`` and ``cylc reload`` to repair a broken workflow
+   configuration without having to run the whole workflow again.
+
+
+.. ifnotslides::
+
+   Consider a workflow where a a large part of the runtime has completed,
+   but a late task has failed.
+
+.. digraph:: workflow
+
+   rankdir="LR"
+   get_data -> process_data
+   get_data[
+      label="get data\n(very slow)",
+      color="forestgreen",
+      fillcolor=palegreen,
+      style=filled
+   ]
+   process_data[
+      label="process data",
+      color="red",
+      fillcolor=palevioletred,
+      style=filled
+   ]
+
+.. ifnotslides::
+
+   Having identified the problem with the late task we want to
+   update the runtime definition of the installed workflow rather than
+   re-run the whole thing.
+
+
+.. practical::
+
+   .. rubric:: In this practical we will break the
+      :ref:`weather-forecasting workflow <tutorial-datetime-cycling-practical>`
+      from the :ref:`scheduling tutorial <tutorial-scheduling>`.
+
+   #. **Get your own copy of the workflow**
+
+      Use ``cylc get-resources`` to get a new copy of the workflow.
+
+
+      .. code-block:: bash
+
+         cylc get-resources tutorial/cylc-forecasting-workflow
+         cd ~/cylc-src/cylc-forecasting-workflow
+
+
+   #. **Break the ``post_process`` task**
+
+      Change the task script for the ``post_process`` task in
+      ``flow.cylc`` to make it fail:
+
+      .. code-block:: diff
+
+         - script = post-process $CYLC_TASK_PARAM_site 60
+         + script = exit 1
+
+   #. **Install, Play and Inspect the workflow**
+
+      And check that ``post_process`` fails.
+
+      .. spoiler:: Step-by-step
+
+         .. code-block:: bash
+
+            cylc validate   # always good practice
+            cylc install
+            cylc play cylc-forecasting-workflow
+            cylc cat-log cylc-forecasting-workflow
```

Comment:
```suggestion
            # Either view in TUI or GUI, or run:
            cylc cat-log cylc-forecasting-workflow
            # Inspect job log for failed task:
```

Reply:
`cylc cat-log` has been renamed `cylc log`. Maybe you could do a global search-and-replace for that, in this PR?.